README.md

```diff
@@ -1 +1,55 @@
-# JS-weather-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Weather App
+
+> This website displays the basic weather data from any city you want.
+
+![screenshot](https://icon-library.com/images/weather-icon-gif/weather-icon-gif-22.jpg)
+
+This site was created using the open weather API for the data and the unsplash API for the images. The idea is to get used to the interaction with APIs to populate information on your pages. This is the fourth project in the Microvese technical curriculum for JavaScript.
+
+## Built With
+
+- Javascript
+- DOM
+- Webpack
+- API
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/mateomh/JS-weather-app/weather-app/dist/index.html)
+
+
+### Usage
+
+Everything you need to run the page is inside the **dist** folder of the project.
```

Comment:
[REQUIRED] Please be more specific about what do we need to do with the files in the dist folder

![image](https://user-images.githubusercontent.com/73254040/96967732-911b3100-14d5-11eb-9826-834a39d2893d.png)
[source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)